src/analysis/calculus/deriv.lean

```diff
@@ -565,8 +572,102 @@ end
 
 end composition
 
-/- Multiplication -/
+section composition_vector
+/-! ###
+Derivative of the composition of a function between vector spaces and of a function defined on `𝕜`
```

Comment:
This should be on the same line as the `###` to render correctly in markdown.

Reply:
Yes, thanks!